dotnet/dynamodb/UpdateItemDataModel/UpdateItemDataModel.cs

```diff
@@ -0,0 +1,170 @@
+            await context.SaveAsync(orderRetrieved);
+
+            // Retrieve the updated item
+            Entry updatedOrder = await context.LoadAsync<Entry>(id, "Order", new DynamoDBOperationConfig
+            {
+                ConsistentRead = true,
+            },
+            new System.Threading.CancellationToken());
+
+            return updatedOrder;
+        }
+
+        static void Main(string[] args)
+        {
+            var configfile = "app.config";
+            var region = "";
+            var table = "";
+            var id = "";
+            var status = "";
+
+            // Get default region and table from config file
+            var efm = new ExeConfigurationFileMap
+            {
+                ExeConfigFilename = configfile
+            };
+
+            Configuration configuration = ConfigurationManager.OpenMappedExeConfiguration(efm, ConfigurationUserLevel.None);
+
+            if (configuration.HasFile)
+            {
+                AppSettingsSection appSettings = configuration.AppSettings;
+                region = appSettings.Settings["Region"].Value;
+                table = appSettings.Settings["Table"].Value;
+
+                if ((region == "") || (table == ""))
+                {
+                    Console.WriteLine("You must specify a Region and Table value in " + configfile);
+                    return;
+                }
+            }
+            else
+            {
+                Console.WriteLine("Could not find " + configfile);
+                return;
+            }
+
+            int i = 0;
+            while (i < args.Length)
+            {
+                switch (args[i])
+                {
+                    case "-i":
+                        i++;
+                        id = args[i];
+                        break;
+                    case "-s":
+                        i++;
+                        status = args[i];
+                        break;
+                    default:
+                        break;
+                }
+
+                i++;
+            }
+
+            if ((status == "") || (id == "") || ((status != "backordered") && (status != "delivered") && (status != "delivering") && (status != "pending")))
+            {
+                Console.WriteLine("You must supply a partition number (-i ID), and status value (-s STATUS) of backordered, delivered, delivering, or pending");
+                return;
+            }
+
+            var newRegion = RegionEndpoint.GetBySystemName(region);
+            var client = new AmazonDynamoDBClient(newRegion);
+            var context = new DynamoDBContext(client);
+
+            var response = UpdateTableItemAsync(context, id, status);
+
+            // Updates status regarless of whether id identifies an order
+            if (response.Result.Order_Status == status)
+            {
+                Console.WriteLine("Successfully updated item in " + table + " in region " + region);
```

Comment:
region -> Region

Reply:
I've trimmed the passage to just "Successfully update the item."